pyproject.toml

```diff
@@ -16,6 +16,7 @@ PyYAML = "^6.0"
 Jinja2 = "^3.1.2"
 python-on-whales = "^0.46.0"
 tenacity = "^8.0.1"
+xgboost = "^1.6.1"
```

Comment:
This should go to dev dependencies, we don't want the base truss package to have many dependencies.